podsixnet2/endpoint.py

```diff
@@ -1,63 +1,66 @@
 # coding=utf-8
 import socket
 
-from podsixnet2.asyncwrapper import poll
-from podsixnet2.Channel import Channel
+from PodSixNet.asyncwrapper import poll
+from PodSixNet.channel import Channel
 
 class EndPoint(Channel):
     """
     The endpoint queues up all network events for other classes to read.
     """
     def __init__(self, address=("127.0.0.1", 31425), map=None):
```

Comment:
Maybe I'm just stupid, but why does the "host" and the "port" have default values (I mean, I understand the "host" but what is the reason for the "port")